pkg/controllers/scheduler/framework/plugins/clusterresources/most_allocated.go

```diff
@@ -49,19 +51,37 @@ func (pl *ClusterResourcesMostAllocated) Score(
 		return 0, framework.NewResult(framework.Error, err.Error())
 	}
 
-	requested := make(framework.ResourceToValueMap, len(framework.DefaultRequestedRatioResources))
-	allocatable := make(framework.ResourceToValueMap, len(framework.DefaultRequestedRatioResources))
+	resources := make([]corev1.ResourceName, 0, len(framework.DefaultRequestedRatioResources))
 	for resource := range framework.DefaultRequestedRatioResources {
+		switch resource {
```

Comment:
We have the same code in three places, can we extract the code to a function?

Reply:
Done.